FunctionsSdkE2ETests/PreservedDependencies.sln

```diff
@@ -0,0 +1,25 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
i don't think you need this solution file, maybe?

Reply:
So I added a solution file because it looks like it's the standard way of establishing a `workingDir` for all the tests (using `FindContainingDirectory()`). I'm sure there's another way to do it if that would be better.

Reply:
Nope you're probably right! I forgot how I did this :)